corona-warn-app/cwa-app-android
# Fix: Duration input layout (DEV)

Quick fix for Duration picker input field.

How to test:
1) Open application
2) Create new location in Contact Diary
3) Check the duration input field
4) Go to device settings and change font size to bigger one
5) Go back to contact diary
6) Check that duration input field is still in one line

![Screenshot 2021-03-11 at 17 52 46](https://user-images.githubusercontent.com/64849422/110807305-f197e880-8293-11eb-8e5f-ed6653101b0f.png)
